tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
Why this? There is nothing in this file that depends on whether or not fenv.h is present, or IEEE754 checks are supported.